app/javascript/controllers/select_user_rows_controller.js

```diff
@@ -0,0 +1,118 @@
+import { Controller } from "@hotwired/stimulus";
+import Rails from "@rails/ujs";
+import Cookies from "js-cookie";
+import DOMPurify from "dompurify";
+
+export default class extends Controller {
+  static targets = ["checkbox", "invitationFormatOption"]
+
+  connect() {
+    /* eslint-disable prefer-destructuring */
```

Comment:
Faudrait peut-être changer la règle car j'avoue que destructurer pour mettre sur this me semble douteux ^^

Reply:
Oui y a pas de moyen de disable la règle juste pour ces cas malheureusement.